ui/components/app/transaction-list-item-details/transaction-list-item-details.component.js

```diff
@@ -135,24 +135,23 @@ export default class TransactionListItemDetails extends PureComponent {
     });
   };
 
-  async componentDidMount() {
+  componentDidMount() {
     const {
       recipientAddress,
       tryReverseResolveAddress,
       ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
-      getCustodianTransactionDeepLink,
       selectedIdentity,
       transactionGroup,
       ///: END:ONLY_INCLUDE_IN
     } = this.props;
 
     ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
-    if (selectedIdentity && transactionGroup?.primaryTransaction?.custodyId) {
-      const custodyTransactionDeepLink = await getCustodianTransactionDeepLink(
-        selectedIdentity.address,
-        transactionGroup.primaryTransaction.custodyId,
-      );
-      this.setState({ custodyTransactionDeepLink });
+    this._mounted = true;
+    const { address } = selectedIdentity;
```

Comment:
Could `selectedIdentity` be undefined?

Reply:
You're right, it can, I'll change this 👍🏼